StatusBar/StatusBar.js

```diff
@@ -40,13 +45,15 @@ class StatusBar extends Component<Props> {
           barStyle={light ? 'light-content' : 'dark-content'}
           backgroundColor={backgroundColor}
         />
-        <Text
-          allowFontScaling={false}
-          style={[styles.title, { color: light ? 'white' : 'black' }]}
-        >
-          {title}
-        </Text>
-        {children}
+        <View style={styles.body}>
+          <Text
```

Comment:
{title && <....>}

Reply:
we will have to wrap it in another view as space-inbetween applies for this as well, View with center will do it, is there other better solution?